<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LoginView</class>
 <widget class="QDialog" name="LoginView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>588</width>
    <height>433</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="Title">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>10</y>
     <width>101</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>云加密</string>
   </property>
  </widget>
  <widget class="QLabel" name="UserNameLabel">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>130</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>用户名</string>
   </property>
  </widget>
  <widget class="QLabel" name="PassWordLabel">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>200</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>密码</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="UserName">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>130</y>
     <width>241</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="PassWord">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>200</y>
     <width>241</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="Register">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>280</y>
     <width>60</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>注册账号</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="RememberPwd">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>280</y>
     <width>87</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>记住密码</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="FindPwd">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>280</y>
     <width>87</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>找回密码</string>
   </property>
  </widget>
  <widget class="QPushButton" name="LoginBtn">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>310</y>
     <width>361</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>登录</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
